fe/fe-core/src/main/java/org/apache/doris/planner/StreamLoadScanNode.java

```diff
@@ -66,8 +66,8 @@ public class StreamLoadScanNode extends LoadScanNode {
     private TupleDescriptor srcTupleDesc;
     private TBrokerScanRange brokerScanRange;
 
-    private Map<String, SlotDescriptor> slotDescByName = Maps.newHashMap();
-    private Map<String, Expr> exprsByName = Maps.newHashMap();
+    private Map<String, SlotDescriptor> slotDescByName = Maps.newTreeMap(String.CASE_INSENSITIVE_ORDER);
```

Comment:
Better add comment to explain why we need CASE_INSENSITIVE_ORDER

Reply:
For example, the column name 「A」 in the table and the mapping '(a) set (A = a)' in load sql， Slotdescbyname stores「a」, If Slotdescbyname case sensitive, later will use 「a」to get table's 「A」 column info, will throw exception.

Reply:
What I mean is it would be better to add this comment in source file, so that other developer can understand it.

Reply:
done